aptos-move/move-examples/marketplace/sources/coin_listing.move

```diff
@@ -121,6 +123,31 @@ module coin_listing {
         listing
     }
 
+    public entry fun update_fixed_price<CoinType>(
+        seller: &signer,
+        object: Object<Listing>,
+        price: u64,
+    ) acquires FixedPriceListing {
+        let listing_addr = object::object_address(&object);
+
+        assert!(exists<FixedPriceListing<CoinType>>(listing_addr), error::not_found(ENO_LISTING));
+
+        let fixedPriceListing = borrow_global_mut<FixedPriceListing<CoinType>>(listing_addr);
+
+        assert!(fixedPriceListing.price != price, error::invalid_argument(ESAME_PRICE))
```

Comment:
I'd just return early here so we don't emit the event because there's no strong reason to fail this transaction

Reply:
syntax is weird here, why introduce camel casing :scream: